packages/react-components/src/organisms/GoogleCalendar.tsx

```diff
@@ -21,7 +21,11 @@ const GoogleCalendar: React.FC<GoogleCalendarProps> = ({ calendars }) => {
   });
 
   return (
-    <iframe src={iframeSrc.toString()} title="Calendar" css={iframeStyles} />
+    <iframe
+      src={iframeSrc.toString()}
+      title="Google Calendar Iframe"
```

Comment:
I prefer the previous title, this doesn't seem user focused (user doesn't necessarily know what 'iframe' means)

Reply:
The problem is we have an iframe called Google Calendar and an SVG which is also a Google Calendar. Would you prefer embed?

Reply:
If it's just about finding it in a test surely it can also be achieved with a more detailed query using `selector` or so?